### ISSUE TITLE
POST to mitigate fails with TypeError: expected string or buffer

### ISSUE THREAD

**Issue by christiankiller** [blockchain-signaling-system/bloss-core#2]
```
aE3ArEYdV1UQb3JtgGryPsFKMkeyPp1n1Hu3o6VpVKDE0xCeL07B0B6236w==', u'action': u'blackhole', u'subnetwork': u'192.168.40.0/24', u'addresses': [u'192.168.40.3', u'192.168.40.2', u'192.168.40.5', u'192.168.40.4']}
Nov 07 10:12:45 controller400 env[20540]: [POLLEN]parse_attack_report_message, type of message: <type 'dict'>
Nov 07 10:12:45 controller400 env[20540]: Inside hash: 1141870679
Nov 07 10:12:45 controller400 env[20540]: Inside target: 192.168.60.5
Nov 07 10:12:45 controller400 env[20540]: [BLOSS]:attack_report None
Nov 07 10:12:45 controller400 env[20540]: [2018-11-07 10:12:45,547] ERROR in app: Exception on /api/v1.0/mitigate [POST]
Nov 07 10:12:45 controller400 env[20540]: Traceback (most recent call last):
Nov 07 10:12:45 controller400 env[20540]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1982, in wsgi_app
Nov 07 10:12:45 controller400 env[20540]:     response = self.full_dispatch_request()
Nov 07 10:12:45 controller400 env[20540]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1614, in full_dispatch_request
Nov 07 10:12:45 controller400 env[20540]:     rv = self.handle_user_exception(e)
Nov 07 10:12:45 controller400 env[20540]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1517, in handle_user_exception
Nov 07 10:12:45 controller400 env[20540]:     reraise(exc_type, exc_value, tb)
Nov 07 10:12:45 controller400 env[20540]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1612, in full_dispatch_request
Nov 07 10:12:45 controller400 env[20540]:     rv = self.dispatch_request()
Nov 07 10:12:45 controller400 env[20540]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1598, in dispatch_request
Nov 07 10:12:45 controller400 env[20540]:     return self.view_functions[rule.endpoint](**req.view_args)
Nov 07 10:12:45 controller400 env[20540]:   File "/opt/bloss/stalk/api.py", line 22, in mitigate
Nov 07 10:12:45 controller400 env[20540]:     json_data = json.loads(request.get_json(force=True))
Nov 07 10:12:45 controller400 env[20540]:   File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
Nov 07 10:12:45 controller400 env[20540]:     return _default_decoder.decode(s)
Nov 07 10:12:45 controller400 env[20540]:   File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
Nov 07 10:12:45 controller400 env[20540]:     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
Nov 07 10:12:45 controller400 env[20540]: TypeError: expected string or buffer
Nov 07 10:12:45 controller400 env[20540]: EVENT ofp_event->Controller EventOFPFlowStatsReply
Nov 07 10:12:45 controller400 env[20540]: http://172.10.15.20:8086 "POST /write?db=mydb HTTP/1.1" 204 0
Nov 07 10:12:45 controller400 env[20540]: http://172.10.15.20:8086 "POST /write?db=mydb HTTP/1.1" 204 0
Nov 07 10:12:46 controller400 env[20540]: http://172.10.15.17:8545 "POST / HTTP/1.1" 200 298
Nov 07 10:12:46 controller400 env[20540]: Starting new HTTP connection (1): 127.0.0.1
```

**`TypeError: expected string or buffer`**

when posted with data such as: 

```
curl -X POST \
  http://172.10.15.17:6001/api/v1.0/mitigate \
  -H 'cache-control: no-cache' \
  -H 'content-type: application/json' \
  -H 'postman-token: bf0f289e-6a3a-6abe-6171-302e6e9861d1' \
  -d '{"hash": -782564186, "target": "192.168.60.5", "timestamp": "2018-11-07-11:02:00", "action": "blackhole", "subnetwork": "192.168.40.0/24", "add
```

**Comment by christiankiller** [comment]
Curling works with a workaround in the post declaration of flask, however, when posting from Axios (bloss-node) this is returned. 

```
Nov 07 12:31:09 controller400 env[4444]: [2018-11-07 12:31:09,059] ERROR in app: Exception on /api/v1.0/mitigate [POST]
Nov 07 12:31:09 controller400 env[4444]: Traceback (most recent call last):
Nov 07 12:31:09 controller400 env[4444]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1982, in wsgi_app
Nov 07 12:31:09 controller400 env[4444]:     response = self.full_dispatch_request()
Nov 07 12:31:09 controller400 env[4444]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1614, in full_dispatch_request
Nov 07 12:31:09 controller400 env[4444]:     rv = self.handle_user_exception(e)
Nov 07 12:31:09 controller400 env[4444]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1517, in handle_user_exception
Nov 07 12:31:09 controller400 env[4444]:     reraise(exc_type, exc_value, tb)
Nov 07 12:31:09 controller400 env[4444]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1612, in full_dispatch_request
Nov 07 12:31:09 controller400 env[4444]:     rv = self.dispatch_request()
Nov 07 12:31:09 controller400 env[4444]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1598, in dispatch_request
Nov 07 12:31:09 controller400 env[4444]:     return self.view_functions[rule.endpoint](**req.view_args)
Nov 07 12:31:09 controller400 env[4444]:   File "/opt/bloss/stalk/api.py", line 42, in mitigate
Nov 07 12:31:09 controller400 env[4444]:     stalk_controller.block_attackers(attack_report)
Nov 07 12:31:09 controller400 env[4444]:   File "/opt/bloss/stalk/controller.py", line 98, in block_attackers
Nov 07 12:31:09 controller400 env[4444]:     for attacker_address in attack_report.addresses:
Nov 07 12:31:09 controller400 env[4444]: AttributeError: 'NoneType' object has no attribute 'addresses'
```

**Comment by christiankiller** [comment]
Closed by 4472fb6
